.vscode/settings.json

```diff
@@ -11,11 +13,28 @@
     },
     "[typescript]": {
         "editor.defaultFormatter": "esbenp.prettier-vscode",
+        "editor.tabSize": 2,
```

Comment:
Note that we already have the .prettierrc file setting this tab width, so do we need to here? I mean it is the same tab width so I guess it shouldn't really be a problem leaving it here

Reply:
To my knowledge, the formatter runs on save while this enforces it upon tab press in-editor. Minor difference, but I figured why not.

Reply:
Oh you are completely right. That is perfect